src/Components/AnimationEditorPage.js

```diff
@@ -38,19 +100,328 @@ export const AnimationEditorPage = props => {
     }).catch(err => console.log(err));
   };
 
+  const openAnimation = item => {
+    setAnimationTitle(item.title);
+    navigation.setOptions({ headerTitle: item.title });
+    setAnimationTempTitle(item.title);
+    saveAnimationState(item.drill);
+  };
+
+  const createNewAnimation = () => {
+    const emptyAnimation = {
+      positions: [[], []],
+      ids: [],
+      texts: [],
+      background: 'endzone',
+    };
+
+    // Create the title of the new animation
+    const defaultTitle = I18n.t('animationEditor.untitledAnimation');
+    let newTitle = defaultTitle;
+
+    const titleId = props.customeAnimations.findIndex(item => item.title === newTitle);
+
+    // If a saved animation already has the default title
+    if (titleId !== -1) {
+      let counter = 1;
+
+      do {
+        newTitle = defaultTitle + ' (' + counter + ')';
+        counter += 1;
+      } while (props.customeAnimations.findIndex(item => item.title === newTitle) !== -1);
+    }
+
+    saveAnimationState(emptyAnimation);
+    setAnimationTitle(newTitle);
+    setAnimationTempTitle(newTitle);
+    setModalManagerVisible(false);
+    navigation.setOptions({ headerTitle: newTitle });
+  };
+
+  const deleteAnimation = item => {
+    // Envoyer l'action de suppression
+    props.deleteAnimation(item.title);
+
+    if (item.title === animationTitle) createNewAnimation();
+  };
+
   return (
     <View style={styles.animationEditorPage}>
-      <AnimationEditor onAnimationChange={saveAnimationState} />
+      <AnimationEditor onAnimationChange={saveAnimationState} animation={currentAnimationState} />
+
+      {/* Modal to manage the dirlls */}
+      <Modal visible={modalManagerVisible}>
```

Comment:
It feels like we should extract this to a new component, with an explicit name (such as CustomDrillManager or something better).

Maybe just leave the modal here